invenio_assets/__init__.py

```diff
@@ -24,20 +24,38 @@
 
 """Media assets management for Invenio.
 
-First make sure you have Flask application with Click support (meaning
-Flask 0.11+ or alternatively use the Flask-CLI extension):
+Invenio-Assets helps you to integrate webassets, installation of NPM packages,
```

Comment:
"helps you with integration of..., installation of..." OR "helps you to integrate..., to install ..."

Shall we attack English cosmetics like this in this PR, or as part of #28?